composer.json

```diff
@@ -50,12 +49,9 @@
     "behatch/contexts": "^2.6",
     "phpunit/phpunit": "^5.0",
     "sensio/generator-bundle": "^3.0",
-    "symfony/phpunit-bridge": "^3.0",
-    "friendsofphp/php-cs-fixer": "^1.9"
+    "symfony/phpunit-bridge": "^3.0"
   },
   "scripts": {
-    "style:check": "./vendor/bin/php-cs-fixer fix --no-interaction --dry-run --diff -vvvv src/",
```

Comment:
As you suggested in #64 recently, I removed php-cs-fixer in favour of style-ci for [Incipio repo](https://styleci.io/repos/41970601)